### PR TITLE
Properties -> Capabilities (1.19.2)

### PR DESCRIPTION
**Pull request by SiverDX**
**Current state**
- Tested:
  - Chains
  - Siren (+ flute)
  - Frozen
  - Cockatrice Scepter

Unsure about this:

https://github.com/SiverDX/Ice_and_Fire/blob/9a3a89fb70b8c146c460490b2de1760e658a34af/src/main/java/com/github/alexthe666/iceandfire/entity/props/SirenData.java#L24

### CODE REVIEW

**Comment by TheBv** [comment]
Oh yeah I messed up when implementing the Siren stuff apparently.
I'll look into it!

File: src/main/java/com/github/alexthe666/iceandfire/entity/props/SirenData.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+            if (holder.horizontalCollision) {
+                holder.setJumping(true);
+            }
+
+            double motionXAdd = (Math.signum(charmedBy.getX() - holder.getX()) * 0.5D - holder.getDeltaMovement().x) * 0.100000000372529;
+            double motionYAdd = (Math.signum(charmedBy.getY() - holder.getY() + 1) * 0.5D - holder.getDeltaMovement().y) * 0.100000000372529;
+            double motionZAdd = (Math.signum(charmedBy.getZ() - holder.getZ()) * 0.5D - holder.getDeltaMovement().z) * 0.100000000372529;
+
+            holder.setDeltaMovement(holder.getDeltaMovement().add(motionXAdd, motionYAdd, motionZAdd));
```
Comment (TheBv) [inline src/main/java/com/github/alexthe666/iceandfire/entity/props/SirenData.java:87]: Not sure why but this doesn't seem to move the player

**Review by TheBv — COMMENTED** [review]
Looks good everything seems to work except for the player being moved.
I genuinely don't know why that is. I assume it might have to do with the change being overridden at some other part in minecrafts vanilla code; I'm not sure though

File: src/main/java/com/github/alexthe666/iceandfire/entity/props/SirenData.java (hunk 1 above)
Comment (SiverDX) [reply src/main/java/com/github/alexthe666/iceandfire/entity/props/SirenData.java:87]: tested with the fix, works again

**Pull request merged**